examples/events/tests/events.js

```diff
@@ -22,4 +21,41 @@ describe("events", () => {
     assert.ok(event.data.toNumber() === 5);
     assert.ok(event.label === "hello");
   });
+
+  it("Multiple events", async () => {
+    // Sleep so we don't get this transaction has already been processed 
+    await sleep(2000);
```

Comment:
Wonder if there's a nicer way to do this.

Reply:
We'd have to create a new transaction with different instruction data.